src/lib/ThumbnailsSidebar.js

```diff
@@ -311,7 +311,7 @@ class ThumbnailsSidebar {
      * @return {Promise} - promise reolves with the image HTMLElement or null if generation is in progress
      */
     createThumbnailImage(itemIndex) {
-        const cacheEntry = this.thumbnailImageCache.get(itemIndex);
+        const cacheEntry = this.thumbnailImageCache && this.thumbnailImageCache.get(itemIndex);
```

Comment:
I do wonder why this class isn't getting instantiated properly to kick off the constructor. There is a bounded cache on `L52`

Reply:
> I do wonder why this class isn't getting instantiated properly to kick off the constructor. There is a bounded cache on `L52`

The function is used as a callback. Maybe the object has been destroyed before the callback happens.